disasterinfosite/models.py

```diff
@@ -333,80 +333,89 @@ def getRelatedTemplate(self):
     def findSnuggetsForPoint(lat=0, lng=0, merge_deform = True):
         pnt = Point(lng, lat)
 
-######################################################
-# GENERATED CODE GOES HERE
-# DO NOT MANUALLY EDIT CODE IN THIS SECTION - IT WILL BE OVERWRITTEN
-# modelsGeoFilters
```

Comment:
Ugh. I'm not sure how we should handle this.  It seems worth contacting Carson to ask about potential future data additions - I'll do that after leaving this note.  Meanwhile, is there a practical way to do this without removing the generated code?  Perhaps doing some reassignment after it?